draft-krishnan-iab-rfc4052bis.md

```diff
@@ -168,6 +168,11 @@ organizations use a form known as a "liaison statement".  Procedures
 for sending, managing, and responding to liaison statements are
 discussed in {{?RFC4053}}.
 
+Note that communications between organizations have no difference to
```

Comment:
I think this needs to talk about this only from the IETF side because we do not know if this true from the other organizations' side